Engine/EditorUI/Helper/Billboard.h

```diff
@@ -8,11 +8,12 @@
 #include <MathGeoLib.h>
 #include <GL/glew.h>
 
+enum class CoreResource;
+
 class Billboard
 {
 public:
-	Billboard(const std::string& texture_path, float width, float height);
-	~Billboard();
```

Comment:
there is no destructor now for Billboards?

Reply:
There is now a constructor, it looks like this:
```
Billboard(CoreResource quad_sprite, float width, float height);
```

The destructor is the default one